### PR TITLE
NFS: initial cthon04 testsuite and put pynfs in the same folder

### PR DESCRIPTION
**Pull request by SeroSun**
This PR mainly initial cthon04 testsuite, and consider more NFS tests will involve, then make some change in previous NFS test - pynfs, move them in a new test folder. Details:
1. Initial cthon04 test, add it into NFS tests
2. Put previous pynfs in the same folder, and reuse its code
3. Remove folder /pynfs and use /nfs

- Related ticket: https://progress.opensuse.org/issues/93396
- Verification run: 
http://10.67.133.102/tests/327
And this PR doesn't change anything to run pynfs: http://10.67.133.102/tests/323 http://10.67.133.102/tests/324

### CODE REVIEW

**Comment by pevik** [comment]
For libntirpc-devel and libtirpc-devel see https://progress.opensuse.org/projects/qe-kernel/activity?from=2021-08-02 (keep discussion on single place).

File: tests/nfs/install.pm [hunk 1]
```diff
@@ -32,16 +32,39 @@ sub install_dependencies {
...
-    my $rel = get_var('PYNFS_RELEASE');
+sub install_dependencies_cthon04 {
+    my @deps = qw(
+      git
+      gcc
+      make
+      patch
+      nfs-client
+      nfs-kernel-server
+      libntirpc-devel
```
Comment (pevik) [inline tests/nfs/install.pm:43]: as I wrote, I suspect only libtirpc-devel is needed.

**Review by pevik — APPROVED** [review]
Please delete libntirpc-devel and IMHO test will be working also on SLE.

File: tests/nfs/install.pm (hunk 1 above)
Comment (SeroSun) [reply tests/nfs/install.pm]: OK, I removed it as told there. Thanks!

**Comment by SeroSun** [comment]
> Please delete libntirpc-devel and IMHO test will be working also on SLE.

@pevik You are right, remove libntirpc-devel will solve the install issue. I got an error in run part after that solve. Does this failure looks familiar to you? http://10.67.133.102/tests/326#step/run/2

File: tests/nfs/run.pm [hunk 2]
```diff
@@ -32,15 +32,24 @@ sub run {
...
     $self->select_serial_terminal;
 
-    script_run('cd ~/pynfs');
-    server_test_all;
+    if (get_var("PYNFS")) {
+        script_run('cd ~/pynfs');
+        pynfs_server_test_all;
+    }
+    elsif (get_var("CTHON04")) {
+        scritp_run('cd ~/cthon04');
```
Comment (pevik) [inline tests/nfs/run.pm:40]: typo: scritp_run => script_run :)

Comment (SeroSun) [reply tests/nfs/run.pm:40]: I must need a new glasses lol

Comment (SeroSun) [reply tests/nfs/run.pm]: And it works, I updated it in verify run. http://10.67.133.102/tests/327

**Review by pevik — APPROVED** [review]

**Pull request merged**